Userland/Libraries/LibWebView/ViewImplementation.cpp

```diff
@@ -181,6 +181,8 @@ ErrorOr<NonnullRefPtr<WebView::WebContentClient>> ViewImplementation::launch_web
     auto new_client = TRY(adopt_nonnull_ref_or_enomem(new (nothrow) WebView::WebContentClient(move(socket), *this)));
     new_client->set_fd_passing_socket(TRY(Core::LocalSocket::adopt_fd(ui_fd_passing_fd)));
 
+    m_handle = new_client->get_window_handle();
```

Comment:
This only applies to Ladybird - can we move this somewhere that Browser also makes use of this?
For example, we could make this a lazy `Optional<String>` class member, where the getter makes the IPC call if it isn't set:
```c++
String const& ViewImplementation::handle() const {
    if (!m_handle.has_value())
        m_handle = client().get_window_handle();
    return *m_handle;
}

mutable Optional<String> m_handle;
````

Reply:
Look at the `#if` surrounding this method - this code doesn't exist on Serenity, therefore Browser won't be able to use this.

For reference, this function is for launching the WebContent process from Ladybird. This function doesn't exist on Serenity because launching WebContent happens from a centralized service called SystemServer.